<?xml version="1.0"?>

<launch>
  <arg name="ns" default="turtlebot3" />

  <!-- launch RViz -->
  <node name="$(anon rviz)" pkg="rviz" type="rviz" args="-d $(find turtlebot3_ik_solver_exercise)/config/ik_solver.rviz" ns="$(arg ns)" />
</launch>
